fvm/transactionStorageLimiter.go

```diff
@@ -4,84 +4,34 @@ import (
 	"fmt"
 
 	"github.com/onflow/cadence/runtime/common"
-	"github.com/opentracing/opentracing-go"
 
 	errors "github.com/onflow/flow-go/fvm/errors"
-	"github.com/onflow/flow-go/fvm/programs"
-	"github.com/onflow/flow-go/fvm/state"
+	"github.com/onflow/flow-go/model/flow"
 )
 
-type TransactionStorageLimiter struct {
-	// A function to create a function to get storage capacity from an address. This is to make this easily testable.
-	GetStorageCapacityFuncFactory func(
-		vm *VirtualMachine,
-		ctx Context,
-		tp *TransactionProcedure,
-		sth *state.StateHolder,
-		programs *programs.Programs,
-		traceSpan opentracing.Span,
-	) (func(address common.Address) (value uint64, err error), error)
-}
-
-func getStorageCapacityFuncFactory(
-	vm *VirtualMachine,
-	ctx Context,
-	tp *TransactionProcedure,
-	sth *state.StateHolder,
-	programs *programs.Programs,
-	traceSpan opentracing.Span,
-) (func(address common.Address) (value uint64, err error), error) {
-	env := NewTransactionEnvironment(ctx, vm, sth, programs, tp.Transaction, tp.TxIndex, traceSpan)
-	return func(address common.Address) (value uint64, err error) {
-		return env.GetStorageCapacity(common.BytesToAddress(address.Bytes()))
-	}, nil
-}
+type TransactionStorageLimiter struct{}
 
 func NewTransactionStorageLimiter() *TransactionStorageLimiter {
-	return &TransactionStorageLimiter{
-		GetStorageCapacityFuncFactory: getStorageCapacityFuncFactory,
-	}
+	return &TransactionStorageLimiter{}
 }
 
-func (d *TransactionStorageLimiter) Process(
-	vm *VirtualMachine,
-	ctx *Context,
-	tp *TransactionProcedure,
-	sth *state.StateHolder,
-	programs *programs.Programs,
-	traceSpan opentracing.Span,
+func (d *TransactionStorageLimiter) CheckLimits(
+	env Enviornment,
+	addresses []flow.Address,
 ) error {
-	if !ctx.LimitAccountStorage {
+	if !env.Context().LimitAccountStorage {
 		return nil
 	}
 
-	getCapacity, err := d.GetStorageCapacityFuncFactory(vm, *ctx, tp, sth, programs, traceSpan)
-
-	if err != nil {
-		return fmt.Errorf("storage limit check failed: %w", err)
-	}
-
-	accounts := state.NewAccounts(sth)
-
-	addresses := sth.State().UpdatedAddresses()
-
 	for _, address := range addresses {
+		add := common.BytesToAddress(address.Bytes())
```

Comment:
for these repeat type var namings, i usually go for longer, e.g. convertedAddress, rather than shorter.

Reply:
got it, fixed this one at 68a7813